importer/Uploader.py

```diff
@@ -46,11 +46,12 @@ def add_descriptions(self, target_item, descriptions, log):
             desc_content = descriptions[description]['value']
             lang = descriptions[description]['language']
             self.wdstuff.add_description(
-                lang, desc_content, target_item, summary=self.summary)
+                lang, desc_content, target_item)
             if log:
                 t_id = target_item.getID()
-                log.logit(t_id + " ADDED DESCRIPTION " +
-                          lang + " " + desc_content)
+                message = "{} ADDED DESCRIPTION {} {}".format(
+                    t_id, lang, desc_content)
+                log.logit(message)
 
     def add_aliases(self, target_item, aliases, log):
```

Comment:
OK. The downside is that if you run them separately there is nothing preventing you from adding something to Alias which is the same as something in label. I would say it's probably best to use addLabelOrAlias as default (combining them both) and overriding it only in cases where you need to make a distinction.

For those cases you can fix the summary issue by simply adding `summary=edit_summary` to `editAliases`.